htdocs/accountancy/class/bookkeeping.class.php

```diff
@@ -1018,7 +1022,10 @@ public function fetchAll($sortorder = '', $sortfield = '', $limit = 0, $offset =
 				$line->fk_docdet = $obj->fk_docdet;
 				$line->thirdparty_code = $obj->thirdparty_code;
 				$line->subledger_account = $obj->subledger_account;
-				$line->subledger_label = $obj->subledger_label;
+
```

Comment:
A fetch is a DAO method. It must return the content of database, so the ->subledger_label that is stored into table.
Fix must be done on the record that does not correctly make the insert, not during the read.